tfjs-backend-webgpu/src/kernels/matmul_packed_webgpu.ts

```diff
@@ -250,7 +250,11 @@ export class MatMulPackedProgram implements WebGPUProgram {
   getShapeFit(bShape: number[]): boolean[] {
     const tileAOuter = this.workGroupSize[1] * this.workPerThread;
     const tileBOuter = this.workGroupSize[0] * this.workPerThread;
-    const tileInner = tileAOuter > tileBOuter ? tileAOuter : tileBOuter;
+    let tileInner = tileAOuter > tileBOuter ? tileAOuter : tileBOuter;
+    if (this.outputShape[1] === 1) {
+      tileInner *=
+          4;  // for makeMatMulVectorSource, tileSize = gl_WorkGroupSize.x * 4.
```

Comment:
For the `makeMatMulVectorSource` algorithm, `this.workPerThread` is 1. However, in the shader, we access 4 data once and calculate the dot of them. So the tileInner equals `this.workGroupSize[0] * this.workPerThread * 4`.